### PR TITLE
Clean up category page styling

### PR DESCRIPTION
**Pull request by jonalport**

### CODE REVIEW

File: app/assets/stylesheets/comfortable_mexican_sofa/admin/base/_forms.scss [hunk 1]
```diff
@@ -228,3 +228,38 @@ table th select {
...
   width: 100%;
 }
+
+.form-list {
+  list-style: none;
+  padding-left: 40px;
+}
+
+.form-list__item {
+  border-bottom: 1px solid #edf0f0;
```
Comment (benbarnett) [inline app/assets/stylesheets/comfortable_mexican_sofa/admin/base/_forms.scss]: Can this be a colour variable? Is it one of our existing?

File: app/assets/stylesheets/comfortable_mexican_sofa/admin/basic.css.scss [hunk 2]
```diff
@@ -51,6 +51,9 @@ hr {
   border-top-width: 1px;
   border-bottom-width: 0;
 
+  &.roomy {
```
Comment (benbarnett) [inline app/assets/stylesheets/comfortable_mexican_sofa/admin/basic.css.scss]: :+1:

**Comment by benbarnett** [comment]
:+1: aside from one small comment

File: app/assets/stylesheets/comfortable_mexican_sofa/admin/base/_forms.scss (hunk 1 above)
Comment (jonalport) [inline app/assets/stylesheets/comfortable_mexican_sofa/admin/base/_forms.scss]: Yeah, $color-gray-pale!

**Comment by asmega** [comment]
:+1:

**Pull request merged**